template-sample-app/template/src/config/store.js

```diff
@@ -76,6 +76,9 @@ export default function configureAppStore() {
   const reducerManager = createReducerManager(staticReducers);
   store = configureStore({
     reducer: reducerManager.reduce,
+    middleware: [
+      ...getDefaultMiddleware({ immutableCheck: false, serializableCheck: false }),
```

Comment:
Why this?

Reply:
That's because we are passing non-serializable values in the actions. Now `evt` from `onViewportChange`, it's okey if we don't pass serializable values, but we must notify the Redux middleware

Reply:
This probably deserves a bigger discussion, but we did quite a bit of effort at the beginning to avoid this, when we were dealing with stores and layers. Up to this point all params were serializable, as recommended (almost enforced) by redux. I think we should explore an alternative approach to avoid this.

Reply:
This probably deserves a bigger discussion, but we did quite a bit of effort at the beginning to avoid this, when we were dealing with stores and layers. Up to this point all params were serializable, as recommended (almost enforced) by redux. I think we should explore an alternative approach to avoid this.

Reply:
Agree